plugins/CustomDimensions/vue/src/List/List.vue

```diff
@@ -0,0 +1,176 @@
+            <tr v-show="scope.numSlotsUsed === 0 && !isLoading">
+              <td colspan="5">{{ translate('CustomDimensions_NoCustomDimensionConfigured') }}</td>
+            </tr>
+            <tr
+              class="customdimension"
+              v-for="customDimension in sortedCustomDimensionsByScope[scope.value]"
+              :class="customDimension.idcustomdimension"
+              :key="customDimension.idcustomdimension"
+            >
+              <td class="index">{{ customDimension.idcustomdimension }}</td>
+              <td class="name">{{ customDimension.name }}</td>
+              <td
+                class="extractions"
+                v-show="scope.supportsExtractions"
+              >
+                <span :class="{'icon-ok': customDimension.extractions[0]?.pattern}" />
+              </td>
+              <td class="active"><span :class="{'icon-ok': customDimension.active}" /></td>
+              <td class="action">
+                <a
+                  class="table-action icon-edit"
+                  :href="`#?idDimension=${customDimension.idcustomdimension}&scope=${scope.value}`"
+                />
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <div class="tableActionBar">
+          <a
+            value
+            :class="{'disabled': !scope.numSlotsLeft}"
+            v-show="!isLoading"
+            :href="`#?idDimension=0&scope=${scope.value}`"
+          >
+            <span class="icon-add" /> {{ translate('CustomDimensions_ConfigureNewDimension') }}
+            <span class="info">({{ translate(
+              'CustomDimensions_XofYLeft',
+              scope.numSlotsLeft,
+              scope.numSlotsAvailable,
+            ) }})</span>
+          </a>
+        </div>
+      </ContentBlock>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { DeepReadonly, defineComponent } from 'vue';
+import {
+  translate,
+  Matomo,
+  ContentIntro,
+  EnrichedHeadline,
+  ContentBlock,
+  ContentTable,
+} from 'CoreHome';
+import { ucfirst } from '../utilities';
+import CustomDimensionsStore from '../CustomDimensions.store';
+import { CustomDimension } from '../types';
+
+export default defineComponent({
+  name: 'listcustomdimensions',
+  components: {
+    EnrichedHeadline,
+    ContentBlock,
+  },
+  directives: {
+    ContentIntro,
+    ContentTable,
+  },
+  created() {
+    CustomDimensionsStore.fetch();
+  },
+  methods: {
+    ucfirst(s: string) {
+      return ucfirst(s);
+    },
+  },
+  computed: {
+    isLoading(): boolean {
+      return CustomDimensionsStore.isLoading.value;
+    },
+    availableScopes(): (typeof CustomDimensionsStore)['availableScopes']['value'] {
+      return CustomDimensionsStore.availableScopes.value;
+    },
+    contentIntroText(): string {
+      const firstPart = translate(
+        'CustomDimensions_CustomDimensionsIntroNext',
+        '<a target=_blank href=\u0027https://piwik.org/docs/custom-variables\u0027>',
```

Comment:
Just out of curiosity: Is there a special reason for not using a `"`, but the encoded `\u0027`?